docs/Course Wiki/X versus Y/CS374A vs ECE374B.md

```diff
@@ -0,0 +1,12 @@
+# CS374A vs ECE374B
+## Relevant Course Reviews:
+- [CS 374A](../CS%20Course%20Offerings/CS374A.md)
+- [ECE 374B](../ECE%20Course%20Offerings/ECE374B.md)
+
+There are two **idependent** versions of 374 offered at UIUC, section A and section B. Both fulfill the same requirements and work as prerequisites for the same classes. They do however have different lectures, different discussions, different homeworks and exams, and different grading. Both sections are crosslisted as CS/ECE. The A section is run by the CS department so we refer to it as CS 374A. The B section is taught by professors from the ECE department, so we refer to it as ECE 374B.
```

Comment:
"independent" is spelled wrong.
Remove "do": "However, they have different lectures, different discussions, ..."